client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/AbstractConfigurationChildBuilder.java

```diff
@@ -27,6 +27,11 @@ public ServerConfigurationBuilder addServer() {
    }
 
    @Override
+   public ClusterConfigurationBuilder addCluster() {
+      return builder.addCluster();
```

Comment:
Using a name can then be used for logging and also for obtaining information on which cluster we're connected to.